explorer/client/src/components/displaybox/DisplayBox.tsx

```diff
@@ -30,18 +53,29 @@ function DisplayBox({ display }: { display: string }) {
         [setHasFailedToLoad]
     );
 
+    const loadedWithoutAllowedState = hasDisplayLoaded && !imgAllowState;
+
+    let showAutoModNotice =
+        !hasFailedToLoad && hasImgBeenChecked && !imgAllowState;
+
+    if (loadedWithoutAllowedState && hasImgBeenChecked) {
+        display = FALLBACK_IMAGE;
+        showAutoModNotice = true;
+    }
+
     return (
         <div className={styles['display-container']}>
-            {!hasDisplayLoaded && (
+            {!hasDisplayLoaded && !showAutoModNotice && (
                 <div className={styles.imagebox} id="pleaseWaitImage">
-                    Please wait for display to load
+                    image loading...
                 </div>
             )}
-            {hasFailedToLoad ? (
+            {hasFailedToLoad && !showAutoModNotice && (
```

Comment:
```suggestion
            {hasFailedToLoad && (
```

nit: redundant per line 59